src/resources/css/tribe-events-full.css

```diff
@@ -1968,6 +1968,18 @@ body.tribe-theme-twentysixteen .datepicker table {
 /* = Events Retina
 =============================================*/
 
+/* Utility Classes */
+
+.screen-reader-text {
```

Comment:
it's such a common class that the only thing I could see it doing would be duplication.  It comes from the html5 boilerplate and is used in some of the more recent core themes.  I'm not opposed but I'm not sure I see the need.